scripts/lib/install

```diff
@@ -325,7 +325,7 @@ fi
 if [ "$package_system" = apt ]; then
     if ! apt-get install -y \
         puppet git curl wget jq \
-        python3 crudini \
+        python3 python3-yaml crudini \
```

Comment:
We should add this dependency to `zulip::base` as well -- otherwise we may have issues with the upgrade path.

Might be also worth adding a comment that any package listed here generally should also be listed in `zulip::base` for the upgrade code path as well as the `yum` list below, just to help transcribe that bit of knowledge.

Otherwise lgtm!